uefi-raw/src/net.rs

```diff
@@ -76,6 +98,16 @@ impl IpAddress {
             v6: Ipv6Address(ip_addr),
         }
     }
+
+    /// Returns the octets of the union. Without additional context, it is not
+    /// clear whether the octets represent an IPv4 or IPv6 address.
+    ///
+    /// # Safety
+    /// Callers must be sure that all underlying bytes were initialized.
+    #[must_use]
+    pub const unsafe fn octets(&self) -> [u8; 16] {
+        unsafe { self.v6.octets() }
+    }
```

Comment:
I think it might be better not to have an `octets` method on the union, since it'd be easy to use wrong. Users can do `unsafe { addr.v4 }.octets()` or `unsafe { addr.v6 }.octets()`, which is still pretty short and more explicit about intent.

Reply:
Makes sense :) I agree!